### PR TITLE
CLOUDP-122972: Add setup force test

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ [CLOUDP-122972](https://jira.mongodb.org/browse/CLOUDP-122972)

* [patch](https://spruce.mongodb.com/task/mongocli_master_e2e_iteractive_atlas_interactive_e2e_patch_53a4581e94008058ac6e47aa271c3289059796b5_628b639e0ae606217b8710b9_22_05_23_10_42_20/tests?execution=0&sortBy=STATUS&sortDir=ASC)

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: build/ci/evergreen.yml [hunk 1]
```diff
@@ -517,6 +517,44 @@ tasks:
           MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
           MCLI_SERVICE: cloud
           E2E_TAGS: atlas,clusters,m0
+  - name: atlas_interactive_setup_e2e
```
Comment (fmenezes) [inline build/ci/evergreen.yml:520]: I would suggest keeping all atlas interactive tests in one variant `atlas_interactive_e2e` this way more tests would run at once

Comment (blva) [reply build/ci/evergreen.yml:520]: Currently, we have 2 tasks:
One for setup and one for quickstart and both are under the same variant `e2e_iteractive`, is this your suggestion? Or do you mean all of them (quickstart and setup) under the same task as well? If the latter, I can update right away, just wanted to clarify. Thanks!

Comment (fmenezes) [reply build/ci/evergreen.yml:520]: the full story here is if your test is too fast (under 7 min) you will be spending more time waiting for a host than running tests, better to keep slightly bigger test variants

Comment (blva) [reply build/ci/evergreen.yml]: cool! so the second option if I understood correctly. thanks for the extra info

**Review by fmenezes — APPROVED** [review]
LGTM

**Pull request merged**